README.md

```diff
@@ -1,19 +1,74 @@
-# Drools 
+## Prerequisites
 
-## Installing and Running Kakfa Cluster
-- Kafka Cluster on Openshift on fedora
-https://github.com/desmax74/openshift-handbook/blob/master/fedora/kafka.md
-- Kafka Cluster on Openshift on ubuntu
-https://github.com/desmax74/openshift-handbook/blob/master/ubuntu/kafka.md
+- Openshift 3.11 or Minishift
 
+- A Kafka Cluster on Openshift 3.11 with Strimzi https://strimzi.io/
+(tested on Openshift 3.11 and strimzi 0.11.1)
+
+## Creation of Kafka's topics
+Create the kafka topics using the files in the kafka-topics folder, 
+the cluster's name y default is "my-cluster", change accordingly in 
+the yaml files with your cluster's name 
+```sh
+oc create -f events.yaml
+oc create -f control.yaml
+oc create -f snapshot.yaml
+oc create -f kiesessioninfos.yaml
```

Comment:
We should describe better the configuration parameters for each topic. For instance it is mandatory that we have a single partition per topic otherwise the ordering is not guarantee or similar. Maybe we can delegate a additional section for that

Reply:
Added a TODO section

Reply:
Can you please create a Jira ticket too? It will be easier to track

Reply:
Ok